### ISSUE TITLE
Champalimaud employees participate in next hackathon?

### ISSUE THREAD

**Issue by hershaw** [LDSSA/forum#33]
hey party people! we’ve got an interesting situation on our hands! The Champalimaud (or however you spell it) people are wanting to do the hackathon with us! that would mean having some teams on the leaderboard that are not subject to actually winning it. Or we could have another leaderboard separate just for them (as long as it’s not too hard for @hcastilho to implement it).

I personally like having them on the same leaderboard without allowing them to actually place in the final standings. So if they win the top 3 slots, they don’t actually get the prizes. It’s a cool opportunity to allow our students to compete alongside people that are already working in the area which is an aspiration for many of them.

What do the rest of you think?

**Comment by PedroGFonseca** [comment]
Not opposed in theory, however it kind of feels like something we should have offered the other companies that gave us the space. Is this something we intend to offer in future? 

TLDR: not against it, but would rather it were a policy rather than an ad-hoc call

**Comment by hcastilho** [comment]
This means they have to register in the portal and creating those teams by hand.
And in the future filtering those users so they don't appear mixed with the students, or deleting them after the hackathon.

**Comment by ivopbernardo** [comment]
@PedroGFonseca totally. I pitched it to our other partners only lightly - nothing formal or even stated in our value prop so I would not consider that other partners were aware. This was something adhoc asked from Champalimaud because they have recently founded a Data Science club inside Champalimaud and wanted to start have a hands-on experience apart from conferences and talks.

If we decide to go forward with opening it to future partners/next year I believe there are further discussions to have regarding the eligibility when the data comes from the partner itself (example, the partner that lend us the space works with fraud detection, the hackathon problem is about fraud detection and naturally people from the partner will have an advantage in terms of domain knowledge) but those will be other discussions. In this case, this does not apply.

Thank you for the input!

**Comment by hcastilho** [comment]
So, is it ok to delete the users after the hackathon?
Also, do we know how many they will be?

There's no easy way to create ad-hoc teams so I want to have a look at this issue beforehand.

**Comment by hershaw** [comment]
Yeah I think it's fine to include them in the normal leaderboard and then delete then afterward. 

@PedroGFonseca I'm a bit worried that I'm missing something here because I'm quite excited about the idea and think that it's be a great experience for everyone involved, which is a far cry from "I'm not against it". I agree that it should be a policy though and am happy to write one so if you think it's alright, I'll move forward with that. 

For this one well move forward ad-hoc though. 

Gabo and Hugo, is there anything that we need to do to prepare for them?

**Comment by gaborsomogyi** [comment]
What I think we'll need:
- add them to the portal, assign them to separate teams (Hugo does it)
- explain them the process (I guess it's part of each hackathon, but for them it will be the first, so it would be useful to go a bit more into details on how the portal works, how submissions work and all the very basics)

Since the hackathon submission is only about uploading a file regardless of how it was generated they shouldn't have any techical difficulties / special requirements (in theory they could calculate on paper, type the results in notepad and still be able to upload a valid submission)

**Comment by hershaw** [comment]
Do you think we have enough time to figure it out tomorrow morning on the spot? If not no worries, we can figure a workaround

**Comment by gaborsomogyi** [comment]
I'm pretty sure it's possible to do it all tomorrow morning

**Comment by PedroGFonseca** [comment]
Oh, to clarify, I'm for it. I just want to make sure we take the opportunity to make it  a policy and not a one off, that's all :)
